src/hooks/use-page-errors.test.tsx

```diff
@@ -1,3 +1,86 @@
+import { renderHook, act } from "@testing-library/react-hooks";
+import { usePageErrors } from "./use-page-errors";
+import { ResultRecord } from "andculturecode-javascript-core";
+
 describe("usePageErrors", () => {
-    test.skip("TODO - https://github.com/AndcultureCode/AndcultureCode.JavaScript.React/issues/22", () => {});
+    describe("initialized", () => {
+        test("returns empty array", async () => {
+            // Arrange & Act
+            const { result } = renderHook(() => usePageErrors());
+
+            // Assert
+            expect(result.current.pageErrors).toBeEmpty();
+        });
+    });
+
+    describe("setPageErrors", () => {
+        test("when set with string array, pageErrors returns array", async () => {
+            // Arrange
+            const errorMessage = "Error Message";
+            const { result } = renderHook(() => usePageErrors());
+
+            // Act
+            act(() => {
+                result.current.setPageErrors([errorMessage]);
+            });
+
+            // Assert
+            expect(result.current.pageErrors.length).toBe(1);
+            expect(result.current.pageErrors[0]).toBe(errorMessage);
+        });
+    });
+
+    describe("handlePageLoadError", () => {
+        test("when error is string, pageErrors returns array", async () => {
+            // Arrange
+            const errorMessage = "Error Message";
+            const { result } = renderHook(() => usePageErrors());
+
+            // Act
+            act(() => {
+                result.current.handlePageLoadError(errorMessage);
+            });
+
+            // Assert
+            expect(result.current.pageErrors.length).toBe(1);
+            expect(result.current.pageErrors[0]).toBe(errorMessage);
+        });
+
+        test("when error is ResultRecord, pageErrors returns array", async () => {
+            // Arrange
+            const errorMessage = "Error Message";
+            const { result } = renderHook(() => usePageErrors());
+
+            // Act
+            act(() => {
+                const error = {
+                    listErrorMessages: () => [errorMessage],
```

Comment:
Consider creating an actual instance of this record vs. casting a POJO as it. Ideally, we'll probably want to promote the `ResultRecordFactory` from `andculturecode-javascript-core` to `andculturecode-javascript-testing` for reuse, but that doesn't have to happen as part of this.